src/require-strict-assert.spec.ts

```diff
@@ -0,0 +1,138 @@
+        errors: [
+          {
+            message: 'Require strict assertion mode.',
+          },
+        ],
+        output: `import { strict as assert } from 'node:assert';
+               import otherModule1 from 'other-module1';
+               import otherModule2 from 'other-module2';`,
+      },
+      {
+        code: `import assert from 'node:assert';
+               assert.strictEqual(value1, value2);`,
+        errors: [
+          {
+            message: 'Require strict assertion mode.',
+          },
+          {
+            message: 'strict method not required when in strict assertion mode.',
+          },
+        ],
+        output: `import { strict as assert } from 'node:assert';
+               assert.equal(value1, value2);`,
+      },
+      {
+        code: `import { strict as assert } from 'node:assert';
+               assert.strictEqual(value1, value2);`,
+        errors: [
+          {
+            message: 'strict method not required when in strict assertion mode.',
+          },
+        ],
+        output: `import { strict as assert } from 'node:assert';
+               assert.equal(value1, value2);`,
+      },
+      {
+        code: `import { strict as assert } from 'node:assert';
+               assert.deepStrictEqual(obj1, obj2);`,
+        errors: [
+          {
+            message: 'strict method not required when in strict assertion mode.',
+          },
+        ],
+        output: `import { strict as assert } from 'node:assert';
+               assert.deepEqual(obj1, obj2);`,
+      },
+      {
+        code: `import { strict as assert } from 'node:assert';
+               assert.notStrictEqual(value1, value2);`,
+        errors: [
+          {
+            message: 'strict method not required when in strict assertion mode.',
+          },
+        ],
+        output: `import { strict as assert } from 'node:assert';
+               assert.notEqual(value1, value2);`,
+      },
+      {
+        code: `import { strict as assert } from 'node:assert';
+               assert.notDeepStrictEqual(obj1, obj2);`,
+        errors: [
+          {
+            message: 'strict method not required when in strict assertion mode.',
+          },
+        ],
+        output: `import { strict as assert } from 'node:assert';
+               assert.notDeepEqual(obj1, obj2);`,
+      },
+      {
+        code: `import { strict as assert } from 'node:assert';
+               const val1 = 'val2';
+               assert.strict(val1, 'val2');`,
+        errors: [
+          {
+            message: 'strict method not required when in strict assertion mode.',
+          },
+        ],
+        output: `import { strict as assert } from 'node:assert';
+               const val1 = 'val2';
+               assert.equal(val1, 'val2');`,
+      },
+      {
+        code: `import assert from 'node:assert/strict';`,
+        errors: [
+          {
+            message: 'Require strict assertion mode.',
```

Comment:
the message should be something like "Invalid form of strict assertion mode" or something like that